### PR TITLE
feat(organizationUsers): add organization users endpoints

### PR DESCRIPTION
**Pull request by vbeleuta**
add new organization api endpoints
[GET] /organization/orgId/users
[GET] /organization/orgId/users/userId
removed alpha header from getUser (tests will pass when GK change is deployed)

### CODE REVIEW

File: lib/create-organization-api.js [hunk 1]
```diff
@@ -371,9 +374,48 @@ export default function createOrganizationApi ({ http }) {
...
+   * const contentful = require('contentful-management')
+   *
+   * client.getOrganization('<organization_id>')
+   * .then((organization) => organization.getUser('id'))
+   * .then((user) => console.log(user))
+   * .catch(console.error)
+   */
+  function getUser (id) {
+    return http
+      .get('users/' + id, {headers})
```
Comment (burakukula) [inline lib/create-organization-api.js:392]: @vbeleuta out of curiosity - what headers are here?

Comment (vbeleuta) [reply lib/create-organization-api.js:392]: there is a const headers defined at the beginning of the file:
const headers = {'x-contentful-enable-alpha-feature': 'organization-user-management-api'}

**Comment by vbeleuta** [comment]
> looks great, only wonder about typings? 🙏

of course!! thank you for reminding me! will add right away

**Pull request merged**